### DISCUSSION TITLE
Cloudflare worker support

### DISCUSSION THREAD

**Discussion by rafiyq** [DioxusLabs/dioxus#2953] [Q&A]
Is there any cloudflare worker support or integration?

**Comment by awdemos** [comment]
plus 1 on this question.

**Comment by wiseaidev** [comment]
Hello!

I haven't tried Cloudflare workers yet, but you can deploy a dioxus web app to Cloudflare pages by following these steps:

1. Install [Wrangler CLI](https://developers.cloudflare.com/workers/wrangler/get-started/):

   ```sh
   npm i -g wrangler
   ```

1. Login to Cloudflare Account from the CLI:

   ```sh
   wrangler login
   ```

1. Build the web app:

   ```sh
   dx build --release
   ```

1. Publish the web app:
   ```sh
   wrangler pages deploy target/dx/<your-dioxus-project-name>/release/web/public
   ```

This should deploy your static dioxus web app to Cloudflare pages.

Hope this helps!

**Comment by jongiddy** [comment]
[This repo](https://github.com/jongiddy/cf-dioxus) contains some examples of integrating Dioxus with Cloudflare Workers.

I could not get the Dioxus 0.6 `fullstack` feature to build with Workers. I did get server functions working using the original `server_fn` crate.
